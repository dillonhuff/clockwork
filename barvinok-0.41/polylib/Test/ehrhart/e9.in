# Pugh's more elaborate example
# 1 <= i
# j <= n
# 2i <= 3j
#--------------

3 5
1 1 0 0 -1
1 0 -1 1 0
1 -2 3 0 0

1 3
1 1 -1

n
Pugh's example
