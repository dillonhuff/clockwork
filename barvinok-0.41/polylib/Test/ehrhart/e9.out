---------------------------------------
Domain:
         n  -1 >= 0
          1 >= 0

Vertices:
[ 3n/2,  n ]
[ 1,  2/3  ]
[ 1,  n ]

Ehrhart Polynomial:
( 3/4 * n^2 + 1/2 * n + [ 0, -1/4 ]_n )

